example/metro.config.js

```diff
@@ -9,32 +10,41 @@ const modules = Object.keys({
   ...pak.peerDependencies,
 });
 
-module.exports = {
-  projectRoot: __dirname,
-  watchFolders: [root],
+module.exports = async () => {
+  const {
+    resolver: { sourceExts, assetExts },
+  } = await getDefaultConfig();
 
-  // We need to make sure that only one version is loaded for peerDependencies
-  // So we exclusionList them at the root, and alias them to the versions in example's node_modules
-  resolver: {
-    blockList: exclusionList(
-      modules.map(
-        (m) =>
-          new RegExp(`^${escape(path.join(root, 'node_modules', m))}\\/.*$`)
-      )
-    ),
+  return {
+    projectRoot: __dirname,
+    watchFolders: [root],
 
-    extraNodeModules: modules.reduce((acc, name) => {
-      acc[name] = path.join(__dirname, 'node_modules', name);
-      return acc;
-    }, {}),
-  },
+    // We need to make sure that only one version is loaded for peerDependencies
+    // So we exclusionList them at the root, and alias them to the versions in example's node_modules
+    resolver: {
+      assetExts: assetExts.filter((ext) => ext !== 'svg'),
+      sourceExts: [...sourceExts, 'svg'],
+      blockList: exclusionList(
+        modules.map(
+          (m) =>
+            new RegExp(`^${escape(path.join(root, 'node_modules', m))}\\/.*$`)
+        )
+      ),
 
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
-  },
+      extraNodeModules: modules.reduce((acc, name) => {
+        acc[name] = path.join(__dirname, 'node_modules', name);
+        return acc;
+      }, {}),
+    },
+
+    transformer: {
+      babelTransformerPath: require.resolve('react-native-svg-transformer'),
```

Comment:
I put some SVG controls into the example so that you could control the player.  This is only in the example portion of the application, so it won't be shipped with the library.  Do y'all have a different preference for how I could show controls in the example that wouldn't require SVG assets?